pkg/apis/numaflow/v1alpha1/const.go

```diff
@@ -131,6 +131,7 @@ const (
 	DefaultCooldownSeconds          = 90  // Default cooldown seconds after a scaling operation
 	DefaultZeroReplicaSleepSeconds  = 180 // Default sleep time in seconds after scaling down to 0, before peeking
 	DefaultMaxReplicas              = 50  // Default max replicas
+	MaxReplicasLimit                = 100 // The highest number of replicas allowed
```

Comment:
Can we use https://github.com/numaproj/numaflow/blob/ba1f493dc57acd2ba080a8107adbaa6b7c27fa7a/pkg/apis/numaflow/v1alpha1/vertex_types.go#L568?

Reply:
I see. By using `GetMaxReplicas`, if user doesn't specify max, we will use `DefaultMaxReplicas` 50. I am ok with that.

Regarding the `MaxReplicasLimit`, do we have a limit on how many replicas are allowed? The concern is if user specifies a very high max, the rator might break because of sending too many http requests within short period of time.

Reply:
What `too many http requests` do you mean? Live pod check with `HEAD`, or the other one?

Reply:
Live pod check with head

Reply:
I don't think you check up to the `MaxReplicasLimit` right? even if a very high `max` is specified, you only check up to the one is live, right? BTW - if that really happens, then you should enhance the live pod check logic, instead of doing tick triggering, do something similar as you did for count check.

Reply:
Yes. I think we can use tick triggering for now and revisit the decision once we have use case of high replicas. Will address the comment.